src/core.c

```diff
@@ -720,7 +720,9 @@ static bool32 ProcessVramGraphicsCopyQueue(void)
                     if ((graphics->src != 0) && (graphics->dest != 0))
 #endif
                     {
+#if (RENDERER != RENDERER_OPENGL)
```

Comment:
Renderer == software for these too?

Reply:
Here I'd like to keep everything except the memcpy intact, because I don't yet know if it breaks something if the size isn't reduced correctly. As well as INC_GRAPHICS_QUEUE_CURSOR() not being called otherwise.

Reply:
Yeah that's fine, I guess I just meant you could use `RENDER == software` instead of `RENDERER != OPEN_GL`?